contracts/core/Allo.sol

```diff
@@ -423,32 +538,26 @@ contract Allo is Initializable, Ownable, AccessControl {
     function _fundPool(address _token, uint256 _amount, uint256 _poolId, IDistributionStrategy _distributionStrategy)
         internal
     {
```

Comment:
It feels a lot cleaner for this function to take the fee first (or skip if it's 0) and then distribute, rather than using the if else and recreating the funding logic in both clauses. But just personal preference.

Reply:
makes sense
fixed in [690d316](https://github.com/allo-protocol/allo-v2/pull/36/commits/690d316029565766e77a4e464b493337dbfe7664)